dynamic-heatmap/README.md

```diff
@@ -0,0 +1,27 @@
+## Example: Dynamic Heatmap using a 587k points worldwide fires dataset
```

Comment:
Why do we need the word "Dynamic", can't we just say "Heatmap"?

Reply:
@felixpalmer let me know what you think now